source/features/releases-tab.tsx

```diff
@@ -104,7 +104,7 @@ async function init(): Promise<false | void> {
 			releasesTab.setAttribute('aria-current', 'page');
 		}
 
-		select('[data-menu-item="insights-tab"]', repoNavigationBar)!.after(
+		select('.dropdown-divider', repoNavigationBar)!.before(
```

Comment:
For the record, this code seems to be executed **after** `more-dropdown`, so `[data-menu-item="insights-tab"]` no longer exists